internal/proxy/github.go

```diff
@@ -57,6 +57,8 @@ func (p *Proxy) handleGitHubRequest(w http.ResponseWriter, r *http.Request, requ
 		return
 	}
 
+	ns = namespace.Name
```

Comment:
@kishansagathiya do we have any test case for this in place?

Reply:
There are some tests for GitHub webhook requests, but since this mistake was not detected by test, clearly we can do with better or more tests.

Reply:
Can you change https://github.com/fabric8-services/fabric8-jenkins-proxy/blob/140b669a04d1a98a9c75d3161f3822fa34a88f83/internal/proxy/github_test.go#L32  to check for return values?

Reply:
This added in commits that came after this comment.